entries/f/freetrade.io.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Freetrade": {
+    "domain": "freetrade.io",
+    "contact": {
+      "facebook": "freetradeapp",
+      "twitter": "freetrade"
+    },
+    "keywords": [
+      "investing"
+    ]
```

Comment:
```suggestion
    ],
    "regions": [
      "au",
      "de",
      "fr",
      "gb",
      "ie",
      "nl",
      "se"
    ]
```

Reply:
I was worried about adding regions for the site with no 2fa option